Gemfile.lock

```diff
@@ -133,8 +134,8 @@ GEM
       sprockets (~> 2.2.1)
     active_model_serializers (0.8.3)
       activemodel (>= 3.0)
-    activemerchant (1.50.0)
-      activesupport (>= 3.2.14, < 5.0.0)
+    activemerchant (1.63.0)
```

Comment:
which change in `Gemfile` generated this specific upgrade?

Reply:
I think upgrading the Stripe gem. It's not strictly necessary to upgrade it now, but I think it will be useful for the (near) future Stripe development - it has some features to make testing Stripe integration easier (e.g. it includes a mock server you can run locally to respond to Stripe API requests).

Reply:
I was just asking in case we manually upgraded it, if we need a specific version we should add it explicitly to `Gemfile` (even if it's an indirect dependency).
In the case instead of being ugraded while upgrading another gem, it's OK to leave it as it is. Thanks!

Reply:
@enricostano it was manually upgraded, we need at least `1.63.0`. `activemerchant` version is specified by the Spree gem, so the change was made there (in our fork). I will make sure we specify the `activemerchant` requirement in our Gemfile. I guess we will have to deal with this again when merging with Spree upgrade work.